.github/workflows/build_and_test.yml

```diff
@@ -979,6 +979,22 @@ jobs:
       VERSIONS: v14 v15 v16 v17
 
     steps:
+      - uses: docker/login-action@v3
+        with:
+          username: ${{ secrets.NEON_DOCKERHUB_USERNAME }}
+          password: ${{ secrets.NEON_DOCKERHUB_PASSWORD }}
+
```

Comment:
```suggestion
```

We already have this step right after the block you've added

Reply:
Thanks, missed that.